cx-dyn-engines-aws/src/main/java/com/checkmarx/engine/aws/AwsEngines.java

```diff
@@ -163,6 +161,14 @@ public AwsEngines(
 		return dynEngines;
 	}
 
+	/**
+	 * Used for Integration test purposes only
+	 * @param instanceId
+	 */
+	public void stop(String instanceId){
```

Comment:
```suggestion
	void stop(String instanceId){
```
Make is package private if for testing purposes only (I assume from a unit test).